src/properties.ts

```diff
@@ -28,6 +28,8 @@ export const PIWIK_URL = getEnvironmentValue("PIWIK_URL");
 
 export const PIWIK_SITE_ID = getEnvironmentValue("PIWIK_SITE_ID");
 
+export const API_URL  = getEnvironmentValue("API_LOCAL_URL");
```

Comment:
I don't think we need this variable, as the sdk will use the environments API_URL if one is not supplied
https://github.com/companieshouse/ch-sdk-node/blob/master/src/index.ts#L12

Reply:
The createApiClient requires a url as a parameter I'd rather pass in something than pass null.